src/main/webapp/mapScript.js

```diff
@@ -141,6 +141,8 @@ function highlightNearbyLocation(map, placeType, currentLocation) {
       for (var i = 0; i < results.length; i++) {
         createMarker(map, results[i].geometry.location, results[i].place_id);
       }
+      message.innerText = results.length - 1 + ' location(s) found.';
```

Comment:
Is there a reason to subtract 1 here? An array of length 5 will contain 5 results/locations.

Reply:
Oh, thanks. I will correct that